gensim/models/experimental/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .dssm import DSSM  # noqa:F401
+from .drmm_tks import DRMM_TKS  # noqa:F401
```

Comment:
all of this import required an keras -> this will be broken by default. Need to add conditional imports (probably directly to your NN models)

Reply:
Something like the below?:
```
try:
    import keras
except ImportError:
    logger.info("We need keras installed for these models?")

```

Reply:
2 variants:
- imports "in concrete place" (probably not so good for your case) https://github.com/RaRe-Technologies/gensim/blob/309da79584440b0ca87130393f8f85a261b38365/gensim/models/keyedvectors.py#L1438
- something with flag, i.e. 
```python
"""My module"""

try:
    import keras
    IS_KERAS_AVAILABLE = True
except ImportError:
    IS_KERAS_AVAILABLE = True

...

class MyNet:
    def __init__(...):
        if not IS_KERAS_AVAILABLE:
             raise ....
````